cicd/iqe_pod/create_iqe_pod.py

```diff
@@ -53,6 +71,19 @@ def _build_test_conf(env_parser):
     env_name = "clowder_smoke"
     env_conf = conf[env_name] = {}
 
+    env_conf["main"] = {
+        "hostname": "front-end-aggregator-ephemeral-20.apps.c-rh-c-eph.8p0c.p1.openshiftapps.com",
```

Comment:
This `hostname` would need a `oc get route front-end-aggregator -o jsonpath='https://{.spec.host}{"\n"}'` equivalent to update it at runtime.